Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/MergedLogFileTest.cs

```diff
@@ -40,8 +40,8 @@ public void Test20Mb()
 				merged.Property(x => x.EndOfSourceReached).ShouldEventually().BeTrue();
 
 				merged.Count.Should().Be(source.Count);
-				merged.GetValue(LogFileProperties.Size).Should().Be(source.GetValue(LogFileProperties.Size));
-				merged.GetValue(LogFileProperties.StartTimestamp).Should().Be(source.GetValue(LogFileProperties.StartTimestamp));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.